helmfile/charts/prometheus-alerts/templates/rules/daily-checks.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.dailyChecks}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "daily-checks" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: daily-checks
+    rules:
+    - alert: S3BucketContentOver{{.Values.s3BucketPercentLimit}}Percent
+      annotations:
+        description: Over {{.Values.s3BucketPercentLimit}} percent of the total s3bucket size limit is used.
+        #runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-s3eightypercenttotalbucketsize
+        summary: Over {{.Values.s3BucketPercentLimit}} percent of the total s3bucket size limit is passed.
+      expr: |-
+        (
+          sum(max_over_time(s3_objects_size_sum_bytes[1h])) >= ({{ .Values.s3BucketMaxSizeLimit }} * 1000000000) * ({{ .Values.s3BucketPercentLimit }} / 100)
```

Comment:
Is this calculated for each bucket separately or summed together?
If it is calculated separately (e.g. alert when harbor bucket goes over 60% of 1TB), do we then see which bucket it is in the alert?

Reply:
Is it summed together. You could get all the buckets values by making it:

max by (bucket) (max_over_time(s3_objects_size_sum_bytes[1h]))

But getting and calculating on the particular values.. perhaps I can write those out in the message somehow at least in the best case? I see what I can do.